src/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/InteropHelpers.cs

```diff
@@ -10,7 +10,7 @@ namespace Internal.Runtime.CompilerHelpers
     /// <summary>
     /// These methods are used to throw exceptions from generated code.
     /// </summary>
-    internal static class InteropHelpers
+    public static class InteropHelpers
```

Comment:
(SharedAssembly in .NET Native for UWP can make this assumption because we know exactly what modules will be part of it and none of them use InternalsVisibleTo)